.github/workflows/inference.yaml

```diff
@@ -30,5 +30,6 @@ jobs:
           CEPH_KEY_ID: ${{ github.event.client_payload.AWS_ACCESS_KEY_ID  || secrets.AWS_ACCESS_KEY_ID }}
           CEPH_SECRET_KEY: ${{ github.event.client_payload.AWS_SECRET_ACCESS_KEY || secrets.AWS_SECRET_ACCESS_KEY }}
           GITHUB_ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN || secrets.ACCESS_TOKEN}}
+          MODE: ${{ github.event.client_payload.MODE || secrets.MODE}}
```

Comment:
Do we want to set the mode as a secret like this? or can we have it default to "1". And allow the user to define it in their workflow file? This will allow for them two toggle training or not in the same repo. I think.

Reply:
this does make sense, but I was thinking more along the lines of not making 1 as the default option, since the training might be an expensive process we can only do it as and when reqd. And for all other scenarios, we can just run the inference. Please lmk if you think otherwise and we can also add more explanation in the readme.

Reply:
OK, that makes sense. Then do we want to default to "0". I'm Just not sure why it needs to be stored as a secret that's set at the repo level. We want to give users a way to overwrite this value depending on the situation.

Reply:
Sure thing, will add a default for `0` here and add a comment explaining it as well